CoreAardvark/ARKExceptionLogging.m

```diff
@@ -27,11 +27,22 @@
 
 NSUncaughtExceptionHandler *_Nullable ARKPreviousUncaughtExceptionHandler = nil;
 
+NSLock *_Nullable ARKUncaughtExceptionLogDistributorsLock = nil;
+
 NSMutableArray *_Nullable ARKUncaughtExceptionLogDistributors = nil;
 
+NSLock * ARKGetUncaughtExceptionLogDistributorsLock()
+{
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        ARKUncaughtExceptionLogDistributorsLock = [[NSLock alloc] init];
+    });
+    return ARKUncaughtExceptionLogDistributorsLock;
+}
+
 void ARKHandleUncaughtException(NSException *exception)
 {
-    for (ARKLogDistributor *logDistributor in ARKUncaughtExceptionLogDistributors) {
```

Comment:
we should lock around this method too, since the lock should be protecting `ARKUncaughtExceptionLogDistributors` in addition to `ARKPreviousUncaughtExceptionHandler`, right?